### PR TITLE
Remove `Chat` prefix from views that don't need it

### PR DESCRIPTION
**Pull request by nuno-vieira**
# In this PR
Removes the `Chat` prefix from the views that don't need it.

### CODE REVIEW

**Review by skorepak — APPROVED** [review]

File: Sources/StreamChatUI/CommonViews/MessageQuoteView/MessageQuoteView.swift [hunk 1]
```diff
@@ -16,10 +16,10 @@ struct ChatMessageQuoteAvatarAlignment: Equatable {
 }
 
 /// A view that displays a quoted message.
-public typealias ChatMessageQuoteView = _ChatMessageQuoteView<NoExtraData>
+public typealias MessageQuoteView = _MessageQuoteView<NoExtraData>
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/CommonViews/MessageQuoteView/MessageQuoteView.swift:19]: Is this a quote view for a "message" or a `ChatMessage`? 🤔 Maybe the latter?

File: Sources/StreamChatUI/CommonViews/CreateChannelButton/CreateChannelButton.swift [hunk 2]
```diff
@@ -6,7 +6,7 @@ import StreamChat
 import UIKit
 
 /// A Button subclass that should be used for creating new channels.
-open class ChatChannelCreateNewButton: _Button, AppearanceProvider {
+open class CreateChannelButton: _Button, AppearanceProvider {
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/CommonViews/CreateChannelButton/CreateChannelButton.swift:9]: Same here? Is it a button for creating a new "channel" or creating a new `ChatChannel`? 🤔 Probably the former?

**Review by VojtaStavik — APPROVED** [review]
LGTM, just some nit questions. Not sure what are the correct answer there to be honest 🤔

File: Sources/StreamChatUI/CommonViews/MessageQuoteView/MessageQuoteView.swift (hunk 1 above)
Comment (nuno-vieira) [reply Sources/StreamChatUI/CommonViews/MessageQuoteView/MessageQuoteView.swift:19]: Then should it be `ChatMessageQuoteView` or `QuoteChatMessageView` ? This is why having `Chat` prefix in some views makes It harder to name things, eventually the views won't have consistent and will be confusing IMO 🤔

File: Sources/StreamChatUI/CommonViews/CreateChannelButton/CreateChannelButton.swift (hunk 2 above)
Comment (nuno-vieira) [reply Sources/StreamChatUI/CommonViews/CreateChannelButton/CreateChannelButton.swift:9]: In this case, it is clear that it should be `CreateChatChannelButton` right? `ChatChannelCreateButton` does not make sense while reading it

File: Sources/StreamChatUI/CommonViews/MessageQuoteView/MessageQuoteView.swift (hunk 1 above)
Comment (nuno-vieira) [reply Sources/StreamChatUI/CommonViews/MessageQuoteView/MessageQuoteView.swift]: Done ✅

File: Sources/StreamChatUI/CommonViews/CreateChannelButton/CreateChannelButton.swift (hunk 2 above)
Comment (nuno-vieira) [reply Sources/StreamChatUI/CommonViews/CreateChannelButton/CreateChannelButton.swift]: Done ✅

**Pull request merged**